Lesson16/OddNumberTask3.java

```diff
@@ -0,0 +1,23 @@
+package Lesson16;
+
+public class OddNumberTask3 extends Thread{
+    private String name;
+
+    public OddNumberTask3(String name) {
+        super(name);
+    }
+
+    @Override
+    public void run () {
+        this.getOddNumbers();
+    }
+
+    private synchronized void getOddNumbers() {
+        for (int i = 0; i < 10; i++) {
+            if (i % 2 != 0) {
+                System.out.println(this.getName() + ": " + i);
+            }
+        }
```

Comment:
також мінорний комент, можна трішки спростити
```suggestion
        for (int i = 1; i < 10; i += 2) {
                System.out.println(this.getName() + ": " + i);
        }
```

Reply:
Оновлено